module/VuFind/src/VuFindTest/Feature/LiveDatabaseTrait.php

```diff
@@ -73,6 +73,12 @@ public function getLiveTableManager()
                 $config['vufind']['config_reader']
             );
             $container->set(\VuFind\Config\PluginManager::class, $configManager);
+            $localDirs = strlen(trim(LOCAL_OVERRIDE_DIR)) > 0
```

Comment:
Would it be more concise/less redundant to do it like this?

```
$prFactory = new \VuFind\Config\PathResolverFactory();
$container->set(
    \VuFind\Config\PathResolver::class,
    $prFactory($container, \VuFind\Config\PathResolver::class)
);
```

Reply:
Method renamed and feature trait for test added to injecting PathResolver to a container.